src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/PkcsHelpers.cs

```diff
@@ -329,11 +329,11 @@ private static unsafe string ToUpperHexString(ReadOnlySpan<byte> ba)
         {
             fixed (byte* baPtr = ba)
             {
-                return string.Create(ba.Length * 2, (new IntPtr(baPtr), ba.Length), (span, args) =>
+                return string.Create(ba.Length * 2, (Ptr: new IntPtr(baPtr), ba.Length), (span, args) =>
```

Comment:
Why is Ptr: needed on the first  parameter but no name required for the second one?

Reply:
The compiler infers the name of the second one based on the accessed property "Length".  It has nothing to infer on the first one.

Reply:
Ah, clever.